index.html

```diff
@@ -15,18 +15,22 @@ <h1>INTENTION TIMER</h1>
       <h2>New Activity</h2>
       <form>
         <p>Select a category:</p>
-        <button class="study-button activity-buttons"><img src="assets/study.svg" alt="Image of lightbulb and book"><br/>
+        <div class="activity-buttons">
+          <button class="btn-default" id="study-button"><img class="study-img" src="assets/study.svg" alt="Image of lightbulb and book"><br/>
           Study</button>
-        <button class="meditate-button activity-buttons"><img src="assets/meditate.svg" alt="Image of lotus flower"><br/>
+          <button class="btn-default" id="meditate-button"><img class="meditate-img" src="assets/meditate.svg" alt="Image of lotus flower"><br/>
           Meditate</button>
-        <button class="exercise-button activity-buttons"><img src="assets/exercise.svg" alt="Image of shoes"><br/>
+          <button class="btn-default" id="exercise-button"><img class="exercise-img" src="assets/exercise.svg" alt="Image of shoes"><br/>
           Exercise</button>
+        </div>
         <label class="user-goal-prompt">What would you like to accomplish during this time?</label>
         <input class="description-input"/>
         <label class="minutes-prompt">Minutes</label>
         <label class="seconds-prompt">Seconds</label>
```

Comment:
This is great, the added div around input fields and activity buttons is a nice touch, especially when it comes to formatting but using it for event targeting is clever too!